go.mod

```diff
@@ -83,6 +81,7 @@ require (
 	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4 // indirect
 	golang.org/x/net v0.7.0 // indirect
 	golang.org/x/term v0.5.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
```

Comment:
Not strictly needed, but this tricks CI into running the azd legs (so we can validate the go version changes) and also fixes the minor formatting issue which has been bugging me for a while ;-))